COM-2157 Added Storenotification for partialPickup, PickupReady and P…

hrishid opened the pull request:
…ickupReminder

michaelsadlonkibo commented on templates/mobile-notification/pickup-reminder.hypr:
```diff
@@ -0,0 +1,2 @@
+{% comment %} message length should not exceed 139 characters {% endcomment %}
```
Should say 160 since 160 is our max message length.

michaelsadlonkibo commented on templates/mobile-notification/ready-for-pickup.hypr:
```diff
@@ -0,0 +1,2 @@
+{% comment %} message length should not exceed 139 characters {% endcomment %}
```
Should say 160 since 160 is our max message length.

michaelsadlonkibo commented on theme.json:
```diff
@@ -411,6 +411,24 @@
+      "id": "shipment.pickupreminder",
+      "properties": {},
+      "template": "mobile-notification/pickup-reminder",
+      "title": "Curbside Ready"
```
Why is the title `Curbside Ready`? Shouldn't it be `Pickup Reminder`?

michaelsadlonkibo commented on theme.json:
```diff
@@ -411,6 +411,24 @@
+      "id": "shipment.partialpickupready",
+      "properties": {},
+      "template": "mobile-notification/ready-for-pickup",
+      "title": "Pickup Ready"
```
Shouldn't the title be `Partial Pickup Ready`?

michaelsadlonkibo commented on labels/de-DE.json:
```diff
@@ -265,6 +265,8 @@
   "mobileOrderConfirmationMessage": "{0} Bestellbenachrichtigungen. Wir werden Ihnen eine SMS senden, wenn sich Ihr Bestellstatus ändert. Die Meldungsfrequenz kann variieren. Antworte {1}, um dich abzumelden. Es können Nachrichten- und Datentarife gelten",
   "mobileOrderOptInMessage": "Vielen Dank, dass Sie sich für SMS-Benachrichtigungen für Ihre Curbside-Bestellung angemeldet haben. Wir werden Sie benachrichtigen, wenn Ihre Bestellung zur Abholung bereit ist.",
   "mobilePickLocation": "Ramasser / Expédier de:",
+  "mobilePickupReminder":"Rappel, les articles de votre commande {0} sont prêts à être ramassés. Vérifiez la commande à {1}",
```
This file is de-DE which means German. A lot of translations in this file are incorrectly French but they should all be in German.

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -338,6 +338,8 @@
   "mobileOrderOptInMessage": "Thank you for opting in to SMS alerts for your Curbside order. We will notify you when your order is ready for pickup.",
   "mobilePickLocation": "Pickup from/Ship from:",
+  "mobilePickupReminder":"Reminder, Item(s) in your order {0} is ready for pickup. Review the order at {1}",
+  "mobileReadyForPickup":"Item(s) in your order {0} is ready for pickup at {1}. Review the order at {2}.",
```
This text matches the request in the story but there are some minor errors in my opinion. We should confirm the text.

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -338,8 +338,8 @@
-  "mobilePickupReminder":"Reminder, Item(s) in your order {0} is ready for pickup. Review the order at {1}",
-  "mobileReadyForPickup":"Item(s) in your order {0} is ready for pickup at {1}. Review the order at {2}.",
+  "mobilePickupReminder":"Reminder - Items in your order {0} is ready for pickup. Review the order at {1}",
+  "mobileReadyForPickup":"Items in your order {0} is ready for pickup at {1}. Review the order at {2}",
```
`order {0} is ready` should say `order {0} are ready`. Otherwise looks good.
Do the same for both templates. Make sure to update the German, too.

michaelsadlonkibo reviewed (approved):
Changes look good minus the `is` => `are` text. Please make that change then this PR is good to go.

Commit:
COM-2156 Fixed text

COM-2156 Fixed code review changes

COM-2156 Fixed Theme.json and Partial-ready-for-Pickup
